frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class CreateTripWidget extends StatefulWidget {
```

Comment:
Did you mean to link this into `CreateTripPageWidget`? It doesn't seem to be referenced anywhere right now, so you can't see/test the behavior.

Reply:
I forgot to actually delete this widget since it was not being used.